src/Lykke.MatchingEngine.Connector/Tools/TcpConnection.cs

```diff
@@ -59,17 +56,13 @@ public class TcpConnection
             try
```

Comment:
Can this method made sync now?

Reply:
No.  socketNotifier.Disconnect(); is async